subgraphs/uniswap-v2/schema.graphql

```diff
@@ -0,0 +1,464 @@
+  amountUSD: BigDecimal!
+
+  ##### DexAmm-Specific #####
+
+  " The pool involving this transaction "
+  pool: LiquidityPool!
+}
+
+type Withdraw implements Event @entity {
+  " { Transaction hash }-{ Log index }"
+  id: ID!
+
+  " Transaction hash of the transaction that emitted this event "
+  hash: String!
+
+  " Event log index. For transactions that don't emit event, create arbitrary index starting from 0 "
+  logIndex: Int!
+  " The protocol this transaction belongs to "
+  protocol: Protocol!
+  " Address that received tokens "
+  to: String!
+  " Market that tokens are withdrawn from "
+  from: String!
+  blockNumber: BigInt!
+  timestamp: BigInt!
+  " Input tokens of the transaction. E.g. the UNI-LP token "
+  inputTokens: [Token!]!
+  " Output tokens of the transaction. E.g. WETH and USDC from a WETH-USDC pool "
+  outputTokens: Token!
+  " Amount of input tokens in the token's native unit "
+  inputTokenAmounts: [BigDecimal!]!
+  " Amount of output tokens in the token's native unit "
+  outputTokenAmount: BigDecimal!
+  " USD-normalized value of the transaction of the underlying (e.g. sum of tokens withdrawn from a pool) "
+  amountUSD: BigDecimal!
+
+  ##### DexAmm-Specific #####
+
+  " The pool involving this transaction "
+  pool: LiquidityPool!
+}
+
+type Swap implements Event @entity {
+  " { Transaction hash }-{ Log index } "
+  id: ID!
+
+  " Transaction hash of the transaction that emitted this event "
+  hash: String!
+
+  " Event log index. For transactions that don't emit event, create arbitrary index starting from 0 "
+  logIndex: Int!
+  " The protocol this transaction belongs to "
+  protocol: Protocol!
+  " Address that received tokens "
+  to: String!
+  " Market that tokens are withdrawn from "
+  from: String!
+  blockNumber: BigInt!
+  timestamp: BigInt!
+  " Token deposited into pool "
+  tokenIn: Token!
+  " Amount of token deposited into pool in native units "
+  amountIn: BigDecimal!
+  " Amount of token deposited into pool in USD "
+  amountInUSD: BigDecimal!
+  " Token withdrawn from pool "
+  tokenOut: Token!
+  " Amount of token withdrawn from pool in native units "
+  amountOut: BigDecimal!
+  " Amount of token withdrawn from pool in USD "
+  amountOutUSD: BigDecimal!
+  ##### DexAmm-Specific #####
+  " The pool involving this transaction "
+  pool: LiquidityPool!
+}
+
+#Added Entities 
+
+# An account is a unique Ethereum address
+type Account @entity {
+  " Address of the account "
+  id: ID!
+}
+
+type DailyActiveAccount @entity {
+  " { Days since Unix epoch }-{ Address of the account } "
+  id: ID!
+}
+
+type Transaction @entity {
+  id: ID! # txn hash
+  blockNumber: BigInt!
+  timestamp: BigInt!
+  # This is not the reverse of Mint.transaction; it is only used to
+  # track incomplete mints (similar for burns and swaps)
+  mints: [Mint!]
+  burns: [Burn!]
+}
+
+type Mint @entity {
+  # transaction hash + "-" + index in mints Transaction array
```

Comment:
Use double quote comments instead. These will show up in the subgraph playground when users browse the schema